src/libcollections/lib.rs

```diff
@@ -47,14 +47,14 @@
 #![feature(placement_in)]
 #![feature(placement_new_protocol)]
 #![feature(shared)]
+#![feature(slice_get_slice)]
 #![feature(slice_patterns)]
 #![feature(specialization)]
 #![feature(staged_api)]
-#![feature(step_by)]
 #![feature(trusted_len)]
 #![feature(unicode)]
 #![feature(unique)]
-#![feature(slice_get_slice)]
+#![feature(untagged_unions)]
```

Comment:
GitHub wasn't loading all the diffs... *sigh* (sorry). I was excited to see a non-ffi usecase for unions and got a bit confused when searching didn't pick up any.